ob-elasticsearch.el

```diff
@@ -115,7 +119,9 @@ to do that."
     (beginning-of-buffer)
     (es-mark-request-body)
     (let ((output (when mark-active
-                    (es-org-execute-request (cdr (assoc :jq params))))))
+                    (es-org-execute-request
+                     (cdr (assoc :jq params))
+                     (cdr (assoc :tablify params))))))
```

Comment:
Ah, I see. You need it for telling which object to make into a table. Maybe it is better to use jq for that?